cloudinit/distros/freebsd.py

```diff
@@ -59,30 +59,31 @@ def _read_hostname(self, filename, default=None):
     def _write_hostname(self, hostname, filename):
         rhel_util.update_sysconfig_file(filename, {'hostname': hostname})
 
-    def create_group(self, name, members):
-        group_add_cmd = ['pw', '-n', name]
-        if util.is_group(name):
-            LOG.warning("Skipping creation of existing group '%s'", name)
+    def create_group(self, group, members):
```

Comment:
I would suggest not changing the signature of this method from its parent.

Nothing is broken by this currently, but it means that callers can't use keyword arguments (i.e. `create_group(name="...")` works across `Distro` classes today, but would stop working with this change). (Or, more likely, it means that if callers introduce code which uses a keyword argument, we would likely break FreeBSD without noticing.)

(Similarly, we might want to change `members` to have a default value, but that inconsistency isn't introduced by this PR so I don't think we need to do it here.)

Reply:
ooooooof, that rename was my suggestion 😅

Reply:
Oops!

I wouldn't be opposed to renaming the parameter in both places, the real problem is the inconsistency.

Reply:
Yes, good point. I fixed the argument name.